sktime/forecasting/compose/__init__.py

```diff
@@ -13,7 +13,6 @@
     "MultioutputTimeSeriesRegressionForecaster",
     "RecursiveTabularRegressionForecaster",
     "RecursiveTimeSeriesRegressionForecaster",
-    "ReducedForecaster",
```

Comment:
Why do you want to remove this here? 👀

Reply:
I assumed that deprecated stuff don't need to go in.  If this should be kept, then we should also add in ReducedReductionForecaster

Reply:
I want to leave it in for the next release and then remove it afterwards.